source/features/ci-link.tsx

```diff
@@ -38,6 +38,9 @@ void features.add({
 	include: [
 		pageDetect.isRepo
 	],
-	waitForDomReady: false,
+	exclude: [
+		() => select.exists('[aria-label="Cannot fork because repository is empty."]')
```

Comment:
I think this can be added as `isEmptyRepo`, what do you think?

Reply:
Yes